forms-flow-web/src/components/Form/Item/Submission/Item/index.js

```diff
@@ -33,7 +33,8 @@ const Item = React.memo(() => {
     return selectRoot("user", state).roles;
   });
   const applicationStatus = useSelector(
-    (state) => state.applications.applicationDetail?.applicationStatus || ""
+    (state) => state.applications.applicationDetail?.applicationStatus ||
+      state.formDelete.customSubmission?.data?.applicationStatus || ""
```

Comment:
state formDelete is confusing we should write a seperate state name for custom submision  data

Reply:
okay. it was the only point where the custom submission data updated. 
will update it either with a new reducer or with the existing reducer.